tests/test_output.py

```diff
@@ -0,0 +1,15 @@
+"""Check that the CLI prints output in the right format."""
+
+from future.moves.subprocess import getoutput
+
+
+def test_output_sorted():
+    """Ensure that the output is sorted by line & column number."""
+    def extract_line_col(line):
+        pieces = line.split(":")
+        return (int(pieces[1]), int(pieces[2]))  # line number, column number
+
+    output = getoutput("proselint --demo")
```

Comment:
This test works fine locally but fails in CI. I suspect that in CI it's invoking an older installed version of `proselint` instead of the one defined by this revision.